### PR TITLE
Apps 2454 shoppable video sidebar

### PR DESCRIPTION
**Pull request by yaniv-li**

### CODE REVIEW

File: src/video-player.js [hunk 1]
```diff
@@ -520,6 +522,9 @@ class VideoPlayer extends Utils.mixin(Eventable) {
     if (publicId instanceof VideoSource) {
       return this.videojs.cloudinary.source(publicId, options);
     }
+    if (this.videojs.shoppable) {
+      this.videojs.shoppable(this.videojs, options.shoppable);
+    }
```
Comment (tsi) [inline src/video-player.js:527]: @yaniv-li should this be wrapped with a build condition? in `/src/plugins/index.js` you wrap it.

File: src/plugins/cloudinary/models/video-source.js [hunk 2]
```diff
@@ -172,6 +172,9 @@ class VideoSource extends BaseSource {
       if (codecTrans && !hasCodecSrcTrans) {
         opts.transformation.push(codecTrans);
       }
+      if (opts.format === 'auto') {
+        delete opts.format;
+      }
```
Comment (tsi) [inline src/plugins/cloudinary/models/video-source.js:177]: @yaniv-li does that belong here?

File: src/plugins/cld-interactive-video/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import ShoppableWidget from '../../components/shoppable-bar/shoppable-widget';
+
+export default function cldInteractivePlugin(player, options) {
+  if (options) {
+    // eslint-disable-next-line no-new
+    new ShoppableWidget(player, options);
+  }
+
+}
+
```
Comment (tsi) [inline src/plugins/cld-interactive-video/index.js:10]: @yaniv-li is this plugin being used anywhere?

Comment (yaniv-li) [reply src/plugins/cld-interactive-video/index.js:10]: this is the shoppable plugin

File: src/plugins/cloudinary/models/video-source.js (hunk 2 above)
Comment (yaniv-li) [reply src/plugins/cloudinary/models/video-source.js:177]: this is the backend CDN chose best option

File: src/video-player.js (hunk 1 above)
Comment (yaniv-li) [reply src/video-player.js:527]: it can be but it not really required as this checks if the shoppable plugin is loaded

**Pull request merged**